dll/win32/browseui/shellfind/CSearchBar.cpp

```diff
@@ -129,6 +154,11 @@ LRESULT CSearchBar::OnSearchButtonClicked(WORD wNotifyCode, WORD wID, HWND hWndC
     CComHeapPtr<SearchStart> pSearchStart(static_cast<SearchStart *>(CoTaskMemAlloc(sizeof(SearchStart))));
     GetDlgItemText(IDC_SEARCH_FILENAME, pSearchStart->szFileName, _countof(pSearchStart->szFileName));
     GetDlgItemText(IDC_SEARCH_QUERY, pSearchStart->szQuery, _countof(pSearchStart->szQuery));
+
+    pSearchStart->SearchHidden = IsDlgButtonChecked(IDC_SEARCH_HIDDEN);
+
+    TRACE("pSearchStart->SearchHidden is '%d'.\n", pSearchStart->SearchHidden);
```

Comment:
Not sure we need traces for each action in the code (useful for debugging, but when it's clear the code works it's not required).

Reply:
OK, removed.